rosdep/base.yaml

```diff
@@ -4372,6 +4372,18 @@ libmsgsl-dev:
   fedora: [guidelines-support-library-devel]
   gentoo: [dev-cpp/ms-gsl]
   ubuntu: [libmsgsl-dev]
+libmysql++-dev:
+  debian: [libmysql++-dev]
+  ubuntu:
+    focal: [libmysql++-dev]
+    jammy: [libmysql++-dev]
```

Comment:
One more change! Please make the `libmysql++-dev` key say `ubuntu: [libmysql++-dev]`. That way this key will also work on Ubuntu Noble and future Ubuntu releases.

https://packages.ubuntu.com/noble/libmysql++-dev

Reply:
Ah, but could you restore the key `libmysql++3v5` to what it was before?  The `libmysql++3v5` key is only available in focal and jammy.

Reply:
done

Reply:
Not quite. The wrong key is split 🙃

Reply:
Today just isn't going my way, xD